<ui version="4.0" >
 <class>ClosingProjectDialog</class>
 <widget class="QDialog" name="ClosingProjectDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Closing Project</string>
  </property>
  <layout class="QVBoxLayout" >
   <item>
    <widget class="QLabel" name="label" >
     <property name="text" >
      <string>The following documents have been modified:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTreeWidget" name="treeWidget" >
     <column>
      <property name="text" >
       <string>Title</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Full Path</string>
      </property>
     </column>
    </widget>
   </item>
   <item>
    <spacer>
     <property name="orientation" >
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" >
      <size>
       <width>382</width>
       <height>33</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" >
     <item>
      <spacer>
       <property name="orientation" >
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" >
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="button_Save_Selected" >
       <property name="text" >
        <string>&amp;Save Selected</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Discard_All" >
       <property name="text" >
        <string>&amp;Discard All Changes</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="button_Cancel" >
       <property name="text" >
        <string>&amp;Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>button_Save_Selected</sender>
   <signal>clicked()</signal>
   <receiver>ClosingProjectDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>144</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel" >
     <x>179</x>
     <y>238</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>button_Cancel</sender>
   <signal>clicked()</signal>
   <receiver>ClosingProjectDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>337</x>
     <y>268</y>
    </hint>
    <hint type="destinationlabel" >
     <x>328</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
